.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: node_js
 before_install:
   - npm install -g npm@latest
+install:
+  - npm install --global electron
```

Comment:
Now that I'm thinking about it, do we really want to test this on Electron as well in this repository? Since Electron is built on Node, it shouldn't be different anyways (if we stick to whatever Node version Electron has in Etcher). Plus, `etcher-image-write` is indirectly tested in Etcher.

This is starting to feel like considerable and not quite necessary overhead to me.
What do you think?